app/src/main/res/layout/offer_terms_area_button.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.button.MaterialButton xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
```suggestion
<Button xmlns:android="http://schemas.android.com/apk/res/android"
```

Also, let's make this button variant an official part of our theme: `Hedvig.Theme.Button.Outlined.Large` and expose it through a similar theme attribute